### PR TITLE
linux-fslc-imx: upgrade to lf-5.15.5-2.0.0 from NXP

### PR DESCRIPTION
**Pull request by angolini**
Signed-off-by: Daiane Angolini <daiane.angolini@foundries.io>

### CODE REVIEW

**Comment by angolini** [comment]
This is the first draft. Please let me know how I can improve the commit message/recipe as I'm struggling to understand how to generate the commit list to be included

**Comment by otavio** [comment]
@angolini we need to upgrade to the latest 5.15 release. Have you finished it?

File: recipes-kernel/linux/linux-fslc-imx_5.15.bb [hunk 1]
```diff
@@ -70,18 +70,18 @@ include linux-fslc.inc
...
+KBRANCH = "5.15-2.0.x-imx"
+SRCREV = "8fe4fe4ac6ffc1510f98e3776895bbc1c07329a8"
 
 # PV is defined in the base in linux-imx.inc file and uses the LINUX_VERSION definition
 # required by kernel-yocto.bbclass.
 #
 # LINUX_VERSION define should match to the kernel version referenced by SRC_URI and
 # should be updated once patchlevel is merged.
-LINUX_VERSION = "5.15.48"
+LINUX_VERSION = "5.15.40"
```
Comment (thochstein) [inline recipes-kernel/linux/linux-fslc-imx_5.15.bb:81]: 48 -> 40?

File: recipes-kernel/linux/linux-fslc-imx_5.15.bb [hunk 2]
```diff
@@ -28,12 +28,12 @@ Latest stable Kernel patchlevel is applied and maintained by Community."
...
 # 1. Stable (tag or SHA(s))
 # ------------------------------------------------------------------------------
-#    tag: v5.15.48
+#    tag: v5.15.40
 #
 # ------------------------------------------------------------------------------
 # 2. NXP-specific (tag or SHA(s))
 # ------------------------------------------------------------------------------
-#    tag: lf-5.15.5-1.0.0
+#    tag: lf-5.15.5-2.0.0
```
Comment (thochstein) [inline recipes-kernel/linux/linux-fslc-imx_5.15.bb:36]: Should be `5.15.40`.

File: recipes-kernel/linux/linux-fslc-imx_5.15.bb (hunk 1 above)
Comment (angolini) [reply recipes-kernel/linux/linux-fslc-imx_5.15.bb:81]: the branch for 2.0.0 right now is at 40. I'm working to get it on later versions, but wanted to review the PR to test the kernel so far, @thochstein

**Review by otavio — APPROVED** [review]

**Comment by thochstein** [comment]
> This is the first draft. Please let me know how I can improve the commit message/recipe as I'm struggling to understand how to generate the commit list to be included

I'm not an expert on this kernel, but the list of changes doesn't seem right. Did you do a log with a range from old to new `SRCREV`?

```
$ git log --oneline b6597629c15cf4fc99dbcdaa079cb1b4c3e92f2e..f4007cadaec7cab415010a8ef0f408183c9804bb
```

**Comment by angolini** [comment]
> > This is the first draft. Please let me know how I can improve the commit message/recipe as I'm struggling to understand how to generate the commit list to be included
> 
> I'm not an expert on this kernel, but the list of changes doesn't seem right. Did you do a log with a range from old to new `SRCREV`?
> 
> ```
> $ git log --oneline b6597629c15cf4fc99dbcdaa079cb1b4c3e92f2e..f4007cadaec7cab415010a8ef0f408183c9804bb
> ```

@thochstein I've changed the commit log. Do you think it is better now?

**Review by thochstein — APPROVED** [review]

**Comment by angolini** [comment]
@otavio , in case there is something missing, could you, please let me know? (I don't see any pending action item from my side)

**Pull request merged**